Userland/Libraries/LibWeb/Fetch/Infrastructure/HTTP/Bodies.cpp

```diff
@@ -103,6 +104,35 @@ void Body::fully_read(JS::Realm& realm, Web::Fetch::Infrastructure::Body::Proces
         });
 }
 
+// https://fetch.spec.whatwg.org/#body-incrementally-read
+void Body::incrementally_read(ProcessBodyChunkCallback process_body_chunk, ProcessEndOfBodyCallback process_end_of_body, ProcessBodyErrorCallback process_body_error, TaskDestination task_destination)
+{
+    HTML::TemporaryExecutionContext const execution_context { Bindings::host_defined_environment_settings_object(m_stream->realm()), HTML::TemporaryExecutionContext::CallbacksEnabled::Yes };
+
+    VERIFY(task_destination.has<JS::NonnullGCPtr<JS::Object>>());
+    // FIXME: 1. If taskDestination is null, then set taskDestination to the result of starting a new parallel queue.
+    // FIXME: Handle 'parallel queue' task destination
+
+    // 2. Let reader be the result of getting a reader for body’s stream.
+    // NOTE: This operation will not throw an exception. We still handle any allocation errors though.
```

Comment:
This comment should have been deleted too :) or at least the second sentence, as that is not from the spec